### ISSUE TITLE
Choosing an algorithm 

### ISSUE THREAD

**Issue by moh55m55** [interedition/collatex#74]
Hello, 
this could be a silly question. According to the documentation, one can choose between three algorithms:  Dekker, Needleman-Wunsch and MEDITE. I do not know how to specify that in the python collate function. Any idea?

**Comment by djbpitt** [comment]
@moh55m55 @rhdekker Not a silly question at all. The Java version of CollateX lets the user select different collation methods. The Python version supports a single method, so there is no need to specify.

**Comment by moh55m55** [comment]
Thanks. What is the method the python version supports?
I see two methods: EditGraphAligner and  ExperimentalAstarAligner

**Comment by djbpitt** [comment]
@moh55m55 @rhdekker The default in the CollateX Python port is **EditGraphAligner**. You can use **ExperimentalAstarAligner** by specifying `astar=True` as a parameter to the `collate()` function. I haven’t tested the Astar aligner and I don’t know how “experimental” it is, that is, whether it is sufficiently functional to produce useful output.

**Comment by moh55m55** [comment]
Thanks. Is the EditGraphAligner based on the Needleman algorithm?

**Comment by rhdekker** [comment]
@moh55m55 @djbpitt Yes, but with enhancements to lower the cost of the token matching and to align more than two witnesses.

**Comment by moh55m55** [comment]
Thanks, 
What is the alignment score used in EditGraphAligner? the scoring schemes (match, mismatch, and gap)?
